python_modules/dagster/dagster/daemon/controller.py

```diff
@@ -138,7 +140,9 @@ def __init__(
 
         self._daemon_shutdown_event = threading.Event()
 
-        self._logger = get_default_daemon_logger("dagster-daemon")
+        configure_loggers(allow_handler_overrides=allow_handler_overrides)
+
+        self._logger = logging.getLogger("dagster.daemon")
```

Comment:
is it idiomatic for the loggers to always use dot / module formatting?

Reply:
I believe it is recommended/idiomatic; https://docs.python.org/3/library/logging.html#logger-objects
> The logger name hierarchy is analogous to the Python package hierarchy, and identical to it if you organise your loggers on a per-module basis using the recommended construction `logging.getLogger(__name__)`